ci/infrastructure/scripts/infrastructure.sh

```
@@ -218,6 +218,7 @@ defaultSettings() {
   fi
   if [ ! -d "$VS_CI_DIR" ]; then mkdir -p $VS_CI_DIR; fi
   if [ ! -d "$VS_CI_DIR/logs" ]; then mkdir -p $VS_CI_DIR/logs; fi
+  if [ ! -d "$VS_CI_DIR/reports" ]; then mkdir -p $VS_CI_DIR/reports; fi
```

Shellcheck (0.10.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)